server/__init__.py

```diff
@@ -126,15 +126,33 @@
     }
 ]
 # Dict of production models, key: model name, value: tuple (clf, generator_fn, predictor_fn).
+# Changed to just load the defualt model
 production_models = {}
 for model_config in MODEL_CONFIGS:
-    print("Loading: " + model_config["name"])
-    clf = joblib.load(model_config["path"])
+    print("Putting in dictionary: " + model_config["name"])
     gen = model_config["generator"]
     pred = model_config["predictor"]
-    production_models[model_config["name"]] = (clf, gen, pred)
+    if model_config["name"] == DEFAULT_MODEL:
+        print("Loading: " + model_config["name"])
+        clf = joblib.load(model_config["path"])
+        production_models[model_config["name"]] = (clf,gen, pred)
```

Comment:
Save the path here too, so that we don't have to look through all the model configs later to lazy-load the model.

`(clf, gen, pred, path)`